src/coord/rmr/conn.c

```diff
@@ -524,13 +524,12 @@ static int checkTLS(char** client_key, char** client_cert, char** ca_cert, char*
   char* clusterTls = NULL;
   char* tlsPort = NULL;
 
+  // If `tls-cluster` is not set to `yes`, we do not connect to the other nodes
+  // with TLS.
   clusterTls = getRedisConfigValue(ctx, "tls-cluster");
   if (!clusterTls || strcmp(clusterTls, "yes")) {
-    tlsPort = getRedisConfigValue(ctx, "tls-port");
```

Comment:
so previously we checked that `tls-cluster==yes` **OR** `tls-port` is defined and non-zero, and now we don't care about the port configuration and only rely on the `tls-cluster` configuration, right? Consider rephrasing the description to be a bit more clear

Reply:
Right.
But now this is true only for OSS, on Enterprise we stay with the same logic so we won't break.
I buffed up the comments a bit, is that what you meant in "description"?

Reply:
The PR description :). I that we move from **only** checking `tls-port` to **only** checking `tls-cluster`

Reply:
That's exactly what it says 🤔 
anyways, added the update that the change is for OSS only